changelog.d/12028.feature

```diff
@@ -0,0 +1 @@
+Add third party event rules callbacks `check_can_shutdown_room` and `check_can_deactivate_user`.
```

Comment:
```suggestion
Add third-party rules callbacks `check_can_shutdown_room` and `check_can_deactivate_user`.
```

Since that's how we call them in the docs